Zend/zend_execute_API.c

```diff
@@ -171,6 +181,23 @@ void init_executor(void) /* {{{ */
 	EG(vm_interrupt) = 0;
 	EG(timed_out) = 0;
 
+#if defined(ZTS) && defined(HAVE_TIMER_CREATE)
+	struct sigevent sev;
+	sev.sigev_notify = SIGEV_THREAD_ID;
+	sev.sigev_value.sival_ptr = &EG(timer);
+	sev.sigev_signo = SIGIO;
+	sev.sigev_notify_thread_id = (pid_t) syscall(SYS_gettid);
+
+	if (timer_create(CLOCK_THREAD_CPUTIME_ID, &sev, &EG(timer)) != 0) {
```

Comment:
Not critical, but it may be possible to create the timer only once per thread (instead of once per request) by moving this to `zend_new_thread_end_handler`, and replacing the `timer_delete` by a disarming `timer_settimer`

Reply:
Thank you! I was looking for a way to do that. I’ll try that.